switchboard/manager.py

```diff
@@ -214,9 +214,20 @@ def as_request(self, user=None, ip_address=None):
         return MockRequest(user, ip_address)
 
     def version_switch(self, switch):
```

Comment:
I haven't looked at all of the code so switchboard may not have a lot of docstrings but they would be helpful. Just looking at this method out of context I'm not sure what it is supposed to be doing. So please add a docstring.